mapie/multi_label_classification.py

```diff
@@ -122,25 +139,34 @@ class MapieMultiLabelClassifier(BaseEstimator, ClassifierMixin):
      [False  True False]
      [False  True False]]
     """
-    valid_methods_ = ["crc", "rcps"]
+    valid_methods_by_metric_ = {
+        "precision": ["ltt"],
+        "recall": ["rcps", "crc"]
+    }
+    valid_methods = list(chain(*valid_methods_by_metric_.values()))
+    valid_metric_ = list(valid_methods_by_metric_.keys())
     valid_bounds_ = ["hoeffding", "bernstein", "wsr", None]
     lambdas = np.arange(0, 1, 0.01)
     n_lambdas = len(lambdas)
     fit_attributes = [
         "single_estimator_",
         "risks"
     ]
-    sigma_init = .25  # Value given in the paper.
+    sigma_init = 0.25
```

Comment:
SUGGESTION: prefer the previous version.
```suggestion
    sigma_init = .25  # Value given in the paper.
```

Reply:
Done